src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -33,7 +33,7 @@ internal sealed class PETypeParameterSymbol
         private DiagnosticInfo _lazyConstraintsUseSiteErrorInfo = CSDiagnosticInfo.EmptyErrorInfo; // Indicates unknown state.
 
         private GenericParameterAttributes _lazyFlags;
```

Comment:
>private GenericParameterAttributes _lazyFlags; [](start = 8, length = 46)

Please change this declaration back to ```private readonly GenericParameterAttributes _flags;```

Reply:
indeed. Nothing is lazy about the flags here.

---
In reply to: [173883919](https://github.com/dotnet/roslyn/pull/25397#discussion_r173883919) [](ancestors = 173883919)